alivee/utils/requests/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class RequestMethodNotSupportedException(Exception):
+    pass
+
+
+class EmptyAreaInputException(Exception):
```

Comment:
The same consideration with `RequestMethodNotSupportedException`, need to record current city, state and zip